### PR TITLE
WIP: DiscIO Variable Consistency

### PR DESCRIPTION
**Pull request by Stevoisiak**
Trying to standardize the variables in DiscIO. Not completely sure what to do with the pVariables.

This is being done in preparation for more cleanup to the DiscIO code.

### CODE REVIEW

File: Source/Core/Core/HW/DSPLLE/DSPLLETools.cpp [hunk 1]
```diff
@@ -59,14 +59,14 @@ bool DumpDSPCode(const u8 *code_be, int size_in_bytes, u32 crc)
 }
 
 // TODO make this useful :p
-bool DumpCWCode(u32 _Address, u32 _Length)
+bool DumpCWCode(u32 _Address, u32 _length)
```
Comment (JosJuice) [inline Source/Core/Core/HW/DSPLLE/DSPLLETools.cpp]: If you're fixing things like these, you should get rid of the leading _ too.

Comment (Stevoisiak) [inline Source/Core/Core/HW/DSPLLE/DSPLLETools.cpp]: I'd have to make sure there isn't any variable conflict before I do that. I think the underscore is mainly used for external variables. Offhand, I don't know of any overlaps, but I would want to be careful to look out for instances such as [WII_IPC_HLE_Device.h](https://github.com/Stevoisiak/dolphin/blob/VariableConsistency/Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device.h#L44-L52)

**Comment by Stevoisiak** [comment]
I'm starting to wonder how I want to break PR up. Maybe I should take care of the leading _ symbols in one commit, decapitalize variables in another, etc.

**Comment by Stevoisiak** [comment]
I think I extended my scope a bit too far with this.

**Pull request closed without merging**